tools/dds/dds-server/dds-device-broadcaster.cpp

```diff
@@ -53,100 +57,85 @@ dds_server::dds_server()
     , _ctx( "{"
             "\"dds-discovery\" : false"
             "}" )
+    , init_done( false )
+
 {
 }
-bool dds_server::init( DomainId_t domain_id )
+bool dds_device_broadcaster::init()
```

Comment:
Do we still need `init()`? Why do we separate it from the ctor?

Reply:
Changed as discussed